server/api/controllers/forms/post-create-form.js

```diff
@@ -0,0 +1,32 @@
+module.exports = {
+  friendlyName: "Create Form",
+
+  description: "Create a new form",
+
+  inputs: {
+    formToSave: {
+      type: "json",
+      required: true,
+      description: "form saved object",
+    },
+  },
+
+  exits: {
+    success: {
+      description: "New form was created successfully.",
+    },
+    error: {
+      description: "Failed to create new form.",
+    },
+  },
+
+  fn: async function ({ formToSave }, exits) {
+    try {
+      const newForm = await Form.create(formToSave).fetch();
+      return exits.success();
```

Comment:
also, wondering if we need to check the JSON object to make sure there's nothing wrong with it before saving?

Reply:
yea this will be handled in future with validation which isn't currently implemented yet